pgmpy/models/MarkovChain.py

```diff
@@ -0,0 +1,299 @@
+#!/usr/bin/env python3
+import numpy as np
+from pandas import DataFrame
+
+from pgmpy.utils import sample_discrete
+
+
+class MarkovChain(object):
```

Comment:
@pratyakshs No need to inherit from `object` in python3. http://stackoverflow.com/a/9448136/1470009